convert-junit4-to-junit5/src/main/java/jb/ConversionReport.java

```diff
@@ -0,0 +1,101 @@
+package jb;
+
+import jb.convert.ConversionOutcome;
+import jb.convert.ConversionResult;
+import jb.convert.UsedFeature;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import static java.lang.String.format;
+import static java.util.Collections.emptyList;
+import static java.util.stream.Collectors.groupingBy;
+import static java.util.stream.Collectors.toList;
+import static jb.convert.UsedFeature.usedFeatureMatching;
+
+class ConversionReport {
+
+    private final List<ConversionResult> results;
+
+    ConversionReport(List<ConversionResult> results) {
+        this.results = results;
+    }
+
+    String print() {
+        List<String> reportLines = new ArrayList<>();
+
+        reportLines.add("");
+        reportLines.add("Report");
+        reportLines.add("unsupported feature usage");
+        appendUnsupportedFeatureUsage(reportLines);
+
+        reportLines.add(" ");
+        reportLines.add("numbers by outcome");
+        appendNumbersByOutcome(reportLines);
+
+        return String.join("\n", reportLines);
```

Comment:
Please add a note in readme that this is OS dependet (or change it to be OS independent - we can create an issue for that)

Reply:
Using `System.lineSeparator()` now.